client/components/deposits-overview/style.scss

```diff
@@ -13,9 +13,13 @@
 		}
 
 		&__description {
-			font-size: 13px;
```

Comment:
I removed the `font-size` because it wasn't necessary - the default font size is 13px.